src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1286,6 +1284,11 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, BindingDiagnost
             this.LookupSymbolsSimpleName(lookupResult, qualifierOpt, plainName, arity, basesBeingResolved, options, diagnose: true, useSiteInfo: ref useSiteInfo);
             diagnostics.Add(node, useSiteInfo);
 
+            if (lookupResult.Kind == LookupResultKind.Empty && qualifierOpt is TypeSymbol typeSymbol)
```

Comment:
>lookupResult.Kind == LookupResultKind.Empty [](http://example.com/codeflow?start=16&length=43)

Is this true when we found something inaccessible or ambiguous?

Reply:
Good catch. Thanks